.drone.star

```diff
@@ -2276,7 +2277,7 @@ def listRemoteCache():
             },
         },
         "commands": [
-            "if test -f runUnitTestsOnly; then echo 'skipping list-ocis-bin-cache'; else mc alias set s3 $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY && mc find s3/owncloud/web/ocis-build; fi",
+            "if test -f runUnitTestsOnly || test -f runTestsForDocsChangeOnly; then echo 'skipping list-ocis-bin-cache'; else mc alias set s3 $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY && mc find s3/owncloud/web/ocis-build; fi",
```

Comment:
```suggestion
            "if test -f doNotRunAcceptanceTests; then echo 'skipping list-ocis-bin-cache'; else mc alias set s3 $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY && mc find s3/owncloud/web/ocis-build; fi",
```
just a suggestion 🤔  if you can create a file `doNotRunAcceptanceTests` too for the cases where we need to skip acceptance suites, then we do not have to check for two files.

Reply:
`doNotRunAcceptanceTests` would then be needed for `docs only` changes, `unit tests only` changes, and for `combination of docs and unit tests only` changes. But we can to run only docs related pipelines in `docs only` changes, skipping `unit tests` as well, but in `unit tests` only or a combination of `docs and unit tests only`, we want to run `unit tests` as well.

Reply:
There is some refactoring that could be done here. But lets get this merged, and check that it works OK. Then we can adjust as needed.